Polyhedron/demo/Polyhedron/Scene_polygon_soup_item.cpp

```diff
@@ -25,6 +26,8 @@
 #include <CGAL/Polygon_mesh_processing/orient_polygon_soup.h>
 #include <CGAL/Polygon_mesh_processing/orientation.h>
 #include <CGAL/Polygon_mesh_processing/repair.h>
+
+#define CGAL_PMP_REPAIR_POLYGON_SOUP_VERBOSE 1
```

Comment:
Is that wanted?

Reply:
Yes, that triggers the debug info in the repair functions.

Reply:
That was my suggestion for a zero-effort way to at least get a little bit of info on what is happening when you click "repair polygon soup" in the demo (otherwise it's just completely silent and it's difficult to see what was done - or not). Note that I use another macro for "spammy" debug output in `repair_polgyon_soup.h`.

Reply:
Thanks for the explanation.